src/reference/sectionD_references.inc

```diff
@@ -14,7 +14,7 @@
 
 .. _ref_unicode:
 
-4. Unicode 13.0.0 :unicode13_display:`/`
+4. Unicode® 13.0.0 :unicode13_display:`/`
 
```

Comment:
Does this render in LaTeX or do you need to specify it too?

Reply:
Yes, it renders fine.

Reply:
Doesn't appear in the latex.

Reply:
It does for me:

<img width="1088" alt="Screen Shot 2020-04-17 at 9 51 00 PM" src="https://user-images.githubusercontent.com/602265/79556717-91d1a580-80f5-11ea-9dda-f5a3aa0e5ac5.png">

Reply:
Could that be a font issue?

Reply:
Oh no it's there now hadn't updated properly.

Reply:
DOH!!!